test/e2e/common/empty_dir.go

```diff
@@ -35,8 +35,8 @@ const (
 )
 
 var (
-	testImageRootUid    = imageutils.GetE2EImage(imageutils.Mounttest)
-	testImageNonRootUid = imageutils.GetE2EImage(imageutils.MounttestUser)
+	testImageRootUid = imageutils.GetE2EImage(imageutils.Agnhost)
+	nonRootUid       = int64(1001)
```

Comment:
what is the reason for this change?

Reply:
So, we have decided to centralize some images into the agnhost image, and to remove images that are no longer very useful. See entire thread here: https://github.com/kubernetes/kubernetes/issues/76342

Among these images is ``mounttest-user``. Looking through the git history, this image was introduced a loooong time ago, before ``PodSecurityPolicy`` was a thing, and thus, you couldn't create pods and tell them to run as certain users; that had to be done in the image itself, and that's exactly what ``mounttest-user`` image's purpose was.

Nowadays, that is no longer needed, we can do that programatically, as you already know. As for why ``nonRootUid`` is ``1001``, that's because the ``mounttest-user`` image was running as UID ``1001``, so I'm just maintaining the behaviour.